zk-token-sdk/src/zk_token_elgamal/pod/mod.rs

```diff
@@ -73,3 +82,27 @@ impl TryFrom<PodProofType> for ProofType {
 #[derive(Clone, Copy, Pod, Zeroable, PartialEq, Eq)]
 #[repr(transparent)]
 pub struct CompressedRistretto(pub [u8; 32]);
+
+macro_rules! impl_from_str {
+    (TYPE = $type:ident, BYTES_LEN = $bytes_len:expr, BASE64_LEN = $base64_len:expr) => {
+        impl FromStr for $type {
```

Comment:
nit: can this be changed to avoid having all the users also import `FromStr`?
```suggestion
        impl std::str::FromStr for $type {
```

Reply:
Oh right, good idea!